src/main.py

```diff
@@ -177,44 +171,60 @@ async def image(interaction: Interaction) -> None:
 @application_checks.has_role(config.dj_role_name)
 async def image_upload(interaction: Interaction, image_file: Attachment) -> None:
     """Upload a Google Forms image as attachment."""
-    global loaded_image
     # TODO: Some basic validity filtering
-    loaded_image.set(image_file.url)
+    # Persist the image URL
+    if not await persistent_state.save_form_image_url(interaction, image_file.url):
+        return
+
+    # Now load it again to ensure that it has saved correctly
+    loaded_image_url = persistent_state.get_form_image_url(interaction)
+
     await interaction.response.send_message(
-        f"\N{WHITE HEAVY CHECK MARK} Image set to: {loaded_image.get()}"
+        f"\N{WHITE HEAVY CHECK MARK} Image set to: {loaded_image_url}"
     )
 
 
 @image.subcommand(name="url")
 @application_checks.has_role(config.dj_role_name)
-async def image_url(interaction: Interaction, image_url: str) -> None:
+async def image_by_url(interaction: Interaction, image_url: str) -> None:
```

Comment:
I renamed this function as technically `image_url` in the function's arguments overwrites the symbol `image_url`. So it'd technically be impossible to access a reference to the function before this change.

(Mostly I just did it because my IDE was yelling at me.)